include/bouncer.h

```diff
@@ -32,6 +32,10 @@
 #include <event2/event.h>
 #include <event2/event_struct.h>
 
+#include "common/postgres_compat.h"
+#include "common/cryptohash.h"
+#include "common/scram-common.h"
```

Comment:
This is importing a lot of very specific stuff into a very general header file. We should find a better way.

The header file `postgres_compat.h` is supposed to make PostgreSQL code compilable in PgBouncer, so I don't see what that is doing here.

The other includes can probably be avoided with some judicious use of forward declarations.

Reply:
Addresesed as best as I could `cryptohash.h` is needed for its enum definition, which I'd rather not forward declare by fully copying it. `scram-common.h` I left out out by hardcoding 32 in all places in this header. `postgres_compat.h` was needed for `cryptohash.h`, but simply defining `uint8` was enough to make this pass.